### ISSUE TITLE
Update to Rimworld 1.4 needed

### ISSUE THREAD

**Issue by martinlyra** [Jaxe-Dev/PublisherPlus#10]
The 1.4 preview is out on the `unstable` beta branch on Steam, and whenever I try to use this one version, I load to a black screen. Hinting at plenty fatal errors.

**Comment by Georodin** [comment]
as far as I can tell in the now stable branch of 1.4 Rimworld, PublisherPlus missing various interface elements

**Comment by martinlyra** [comment]
Yeah, when I went to hack somebody else's fork (a slightly older version) to work for 1.4, the code was using an UI element that no longer exists and the "Selected Mod" variable no longer exists. But rest of the code is fine.
